pkg/consensus/consensus.go

```diff
@@ -479,22 +479,12 @@ func (s *service) getVerifiedTx(count int) []block.Transaction {
 	return res
 }
 
-func (s *service) getValidators(txx ...block.Transaction) []crypto.PublicKey {
+func (s *service) getValidators(_ ...block.Transaction) []crypto.PublicKey {
 	var (
 		pKeys []*keys.PublicKey
 		err   error
 	)
-	if len(txx) == 0 {
```

Comment:
When invoked with non-zero sized slice, it is used for calculating `NextConsensus` field.
DBFT needs to be changed, but for now we can return `GetNextBlockValidators` when invoked with non-zero slice.
https://github.com/neo-project/neo/blob/master/src/neo/Consensus/ConsensusContext.cs#L349

Reply:
I don't think DBFT needs any change wrt this. We already do return next block validators from `GetValidators`.